src/app/core/services/rundown-timing.service.ts

```diff
@@ -0,0 +1,125 @@
+import { BackwardRundownTiming, RundownTiming } from '../models/rundown-timing'
+import { RundownTimingType } from '../enums/rundown-timing-type'
+import { Rundown } from '../models/rundown'
+import { Segment } from '../models/segment'
+import { PartEntityService } from './models/part-entity.service'
+import { Part } from '../models/part'
+
+export class RundownTimingService {
+  private readonly partEntityService: PartEntityService = new PartEntityService()
```

Comment:
You are bypassing our dependency injection here. That's a big no no ;) 
Also, `RundownTimingService` is also not exported to be dependency injected.

Reply:
Dependecy injection is now used!